### DISCUSSION TITLE
Store Telemetry Data Per Customer in Signoz

### DISCUSSION THREAD

**Discussion by adugar-conga** [SigNoz/signoz#1972] [Q&A]
Hi 

Can we store telemetry data per customer in signoz ?

thanks
anil dugar

**Comment by pranay01** [comment]
hey @adugar-conga Can you share more details on what you mean. Do you want to store telemetry data for different customers in different shards? Can you also share why are you interested in storing telemetry data in such a way

**Reply by ba400** [reply]
> Can we store telemetry data per customer in signoz ?

**Comment by ba400** [comment]
> Hi 
> 
> 
> 
> Can we store telemetry data per customer in signoz ?
> 
> 
> 
> 
> 
> thanks
> 
> anil dugar
